app/src/main/res/values/colors.xml

```diff
@@ -29,4 +29,9 @@
     <color name="blue_primary_dark">#ff293c68</color>
     <color name="blue_accent">#738ffe</color>
 
+    <color name="swipe_color_1">#33B5E5</color>
```

Comment:
Why the google colors? Wouldn't be using something like blue_accent with blue_primary better looking?